types/src/write_set.rs

```diff
@@ -40,6 +40,21 @@ impl WriteOp {
     }
 }
 
+pub trait DeserializeU128 {
```

Comment:
this needs more context, can't we have it like this?
```
pub trait DeltaValue: Debug + Clone + Eq + Send + Sync {
  fn deserialize(&self) -> Option<u128>;
  fn serialize(&self) -> Vec<u8>;
}
```

Reply:
there is no need for serialize, it already has Vec<u8> internally.
re:deserialize this is write_set file in types, which doesn't include any other crate. I find u128 clearer because this is actually what we do here from abstraction prospective, then delta happens to need u128 and uses that u128. 

Doing the trait inside aptos-move is even worse because we need it for write-op.

Reply:
TransactionWrite and extract_raw_bytes that returns Option<Vec<u8>>